build-artifacts/project-template-gradle/app/build.gradle

```diff
@@ -254,7 +254,7 @@ task addDependenciesFromNativeScriptPlugins {
 			project.dependencies.add("compile", [name: fileName, ext: "aar"])
 		}
 
-		def jarFiles = fileTree(dir: file("${dep.directory}/platforms/android"), include: ["**/*.jar"])
+		def jarFiles = fileTree(dir: file("$rootDir/${dep.directory}/$PLATFORMS_ANDROID"), include: ["**/*.jar"])
```

Comment:
$PLATFORMS_ANDROID is undefined, consider defining it at the top, or returning the previously hard-coded `platforms/android` string.